src/misc_gui.cpp

```diff
@@ -656,7 +656,7 @@ void HideFillingPercent(TextEffectID *te_id)
 }
 
 static const NWidgetPart _nested_tooltips_widgets[] = {
-	NWidget(WWT_PANEL, COLOUR_GREY, WID_TT_BACKGROUND), SetMinimalSize(200, 32), EndContainer(),
+	NWidget(WWT_EMPTY, COLOUR_GREY, WID_TT_BACKGROUND), SetMinimalSize(200, 32), EndContainer(),
```

Comment:
It's it's empty, does it need to be grey? Isn't there an invalid colour to use? (Also, why does there need to be an invisible widget under the tooltip?) Lots of questions, sorry. 😛